### ISSUE TITLE
App ideas

### ISSUE THREAD

**Issue by whytester** [CodeHubOrg/discussions#4]
Just kicking off the new year (2016) app ideas for the team to collaborate on an app project that (hopefully!) helps solve a real world issue and learn JS on the way.  Basic idea I have is building something to help people avoid shopping in large chain shops, maybe by searching for items using the  app then receiving a list/map of local independent suppliers.

**Comment by PaulGrimshaw** [comment]
:+1:  for the idea above ;-)

**Comment by PaulGrimshaw** [comment]
Be nice to make it an open source GitHub project, for others to use too.

**Comment by katjad** [comment]
>>> My preference would be to go for a bare bones system, i.e. build it up from a clean ubuntu instance. Nice to have an understanding of all the component parts, rather than starting a premade stack.

Yes definitely. Also like the idea. We should have a look what similar projects are already out there, and also what data we can use. Will try and do some of that by next meetup.
